### PR TITLE
Annotations

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

**Comment by lukecampbell** [comment]
@DanielJMaher

File: ooiservices/app/__init__.py [hunk 1]
```diff
@@ -85,4 +85,52 @@ def create_app(config_name):
...
+            delete_links.append((url, rule.endpoint))
+    # links is now a list of url, endpoint tuples
+    doc = {
+        'get_links' : get_links,
+        'put_links' : put_links,
+        'post_links' : post_links,
+        'delete_links' : delete_links
+    }
+
+    return jsonify(**doc)
```
Comment (lukecampbell) [inline ooiservices/app/__init__.py]: Really handing for finding what URLs are ACTUALLY registered within the app.

**Comment by DanielJMaher** [comment]
@birdage @FBRTMaka  Can you take 5 and review this?

**Comment by birdage** [comment]
:+1:

**Comment by FBRTMaka** [comment]
:+1: Like it!

**Pull request merged**